### PR TITLE
Add list component and update margins.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
👍

File: CredentialCardDetail.vue [hunk 1]
```diff
@@ -111,6 +166,39 @@ export default {
     },
     buttonLabel() {
       return this.viewMore ? 'View More' : 'View Less';
+    },
+    detailsFields() {
+      const newFields = {...this.fields};
+      for(const [key, value] of Object.entries(newFields)) {
+        for(const [key2, value2] of Object.entries(this.schema)) {
+          console.log(value2);
```
Comment (aljones15) [inline CredentialCardDetail.vue]: do we need still need this logger?

File: CredentialCardDetail.vue [hunk 2]
```diff
@@ -111,6 +166,39 @@ export default {
...
+            newFields[key] = value;
+          }
+        }
+      }
+      return newFields;
+    },
+    presentationSchema() {
+      const newSchema = {};
+      for(const [key, value] of Object.entries(this.schema)) {
+        console.log(key, value);
```
Comment (aljones15) [inline CredentialCardDetail.vue]: same here do we need this logger?

**Review by aljones15 — APPROVED** [review]
I'm approving this, but because this is going into @gannan08 's branch I would wait on him. I made a sister PR with pictures, and a test credential, etc

https://github.com/digitalbazaar/bedrock-vue-credential-card/pull/36

**Comment by aljones15** [comment]
@gannan08 this is good to go into your `support-vanilla-vcs` branch when you are ready.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**